packages/terra-worklist-data-grid/CHANGELOG.md

```diff
@@ -12,8 +14,8 @@
   * Added `overflowColumns` prop for columns that are scrollable.
   * Added support for pinned columns.
   * Added ability to dive into cells with focusable elements.
-  * Added ability to select the row selection column header to allow sorting by consumers.
   * Added ability to treat the `terra-worklist-data-grid` component as a single tabstop.
+  * Added ability to select the row selection column header to allow sorting by consumers.
```

Comment:
I think you need to update your branch with main so GitHub doesn't think you're adding this line.

Reply:
That's fine then.